internal/db/schema/migrations/postgres/2/02_vault_credential.up.sql

```diff
@@ -0,0 +1,214 @@
+begin;
+
+  create table credential_vault_store (
+    public_id wt_public_id primary key,
+    scope_id wt_scope_id not null
+      constraint iam_scope_fk
+        references iam_scope (public_id)
+        on delete cascade
+        on update cascade,
+    name wt_name,
```

Comment:
should name and description be not null?  
https://github.com/hashicorp/boundary/blob/main/internal/db/schema/migrations/postgres/1/02_domains.up.sql#L39

Reply:
Name and description are optional.